awscli/customizations/emr/emrutils.py

```diff
@@ -248,13 +240,13 @@ def call_and_display_response(session, operation_name, parameters,
                               parsed_globals):
         cli_operation_caller = CLIOperationCaller(session)
         cli_operation_caller.invoke(
-            session.get_service('emr').get_operation(operation_name),
+            'emr', xform_name(operation_name),
```

Comment:
I don't think we want the `xform_name` call here. `invoke` uses `xform_name` internally:

```
    def invoke(self, service_name, operation_name, parameters, parsed_globals):
        client = self._session.create_client(
            service_name, region_name=parsed_globals.region,
            endpoint_url=parsed_globals.endpoint_url,
            verify=parsed_globals.verify_ssl)
        py_operation_name = xform_name(operation_name)
```